plasmapy/utils/decorators/tests/test_checks.py

```diff
@@ -217,7 +217,7 @@ def func(x, y, z=10 * u.eV):
         func(1 * u.m, 1 * u.s, z=np.inf * u.eV)
 
 
-class Test_check_quantity_none_shall_pass:
+class TestCheckQuantityNoneShallPass:
```

Comment:
Yeah, I guess that's fair. I didn't really see that aspect of pep8 as relevant to test grouping classes - but sure, why not. :)